content/docs/tasks/security/mtls-migration/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Mutual TLS Migration
+description: Shows you how to migrate your Istio services to mutual TLS incrementally.
+weight: 10
+keywords: [security,authentication,migration]
+---
+
+This task shows how to migrate your existing Istio services' traffic from plain text to mutual TLS.
```

Comment:
Add "without traffic breakage" to this sentence? This is the essence of this permissive idea and we should sell it here.

Reply:
s/how to migrate/how to smoothly migrate

Reply:
Done. Leave out the smoothly since that's essentially what not breaking traffic means.